src/docker-osx-dev

```diff
@@ -871,6 +893,8 @@ function instructions {
   echo -e "  -e, --exclude-path PATH\t\tExclude PATH while syncing. Behaves identically to rsync's --exclude parameter. May be specified multiple times. Default: $DEFAULT_EXCLUDES"
   echo -e "  -c, --compose-file COMPOSE_FILE\tRead in this docker-compose file and sync any volumes specified in it. Default: $DEFAULT_COMPOSE_FILE"
   echo -e "  -i, --ignore-file IGNORE_FILE\t\tRead in this ignore file and exclude any paths within it while syncing (see --exclude). Default: $DEFAULT_IGNORE_FILE"
+  echo -e "  --only-dependencies\t\t\tDuring install, only install the homebrew dependencies. Useful if homebrew is needs to be run as a different user."
```

Comment:
Sure, if you want, but there's only 26 of them, and I figured this usecase was obscure enough not to spend a short option on it.